### PR TITLE
opentelemetry-gke: add self observability dashboards for the OTel Collector

### PR DESCRIPTION
**Pull request by ridwanmsharif**

### CODE REVIEW

File: dashboards/opentelemetry-gke/metadata.yaml
Comment (ridwanmsharif) [reply dashboards/opentelemetry-gke/metadata.yaml]: Changed the display name in the dashboard

File: dashboards/opentelemetry-gke/opentelemetry-collector.json
Comment (ridwanmsharif) [reply dashboards/opentelemetry-gke/opentelemetry-collector.json]: Done.

**Comment by ridwanmsharif** [comment]
I updated the dashboard so it doesn't refer to GKE specifically. The OTel collector can be run with the same self observability metrics on GCE as well.

File: dashboards/opentelemetry-collector/opentelemetry-collector.json
Comment (johnbryan) [inline dashboards/opentelemetry-collector/opentelemetry-collector.json:109]: Would this metric include RPCs to other observability backends too? If so could we give this chart a more generic name?

**Review by johnbryan — APPROVED** [review]
Looks good, just one comment on a chart title

File: dashboards/opentelemetry-collector/opentelemetry-collector.json
Comment (ridwanmsharif) [reply dashboards/opentelemetry-collector/opentelemetry-collector.json:109]: This tracks RPCs only to GCP Cloud Observability

**Comment by ridwanmsharif** [comment]
Thanks for the review @johnbryan , I don't have write access but let me know if you want me to change anything else before you merge the PR

**Comment by damemi** [comment]
@ridwanmsharif could you add a link to this from the readme in https://github.com/GoogleCloudPlatform/otlp-k8s-ingest with a little section on using it with the self-observability metrics in the readme?

**Pull request merged**
